src/main/java/se/diabol/jenkins/pipeline/util/PluginUtil.java

```diff
@@ -0,0 +1,40 @@
+/*
+This file is part of Delivery Pipeline Plugin.
+
+Delivery Pipeline Plugin is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Delivery Pipeline Plugin is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Delivery Pipeline Plugin.
+If not, see <http://www.gnu.org/licenses/>.
+*/
+package se.diabol.jenkins.pipeline.util;
+
+import jenkins.model.Jenkins;
+
+public class PluginUtil {
+
+    /**
+     * Returns whether the specified plug-in is installed.
+     *
+     * @param shortName
+     *            the plug-in to check
+     * @return <code>true</code> if the specified plug-in is installed,
+     *         <code>false</code> if not.
+     */
+    public static boolean isPluginInstalled(final String shortName) {
+        Jenkins instance = Jenkins.getInstance();
+        if (instance != null) {
+            return instance.getPlugin(shortName) != null;
+        }
+        return true;
```

Comment:
If instance is null it means that Jenkins is either initializing or on its way down and if that is the case we cant do that much so throw an IllegalStateException.